src/Parts.elm

```diff
@@ -74,7 +122,7 @@ type alias Update model msg =
 {-| Standard TEA view function type. 
 -}
 type alias View model a = 
-  model -> a
+  model -> Html a
```

Comment:
I think this is overly restrictive. View functions need arguments (e.g., I'm making a button component and I want to have a `disabled` option).